app/views/user/orders/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1 align = "center"><%=@order.id%></h1>
+
+<h1 align = "center">Order Info</h1>
+<center>
+  <table>
+    <tr>
+      <th>Item</th>
+      <th>Sold By</th>
+      <th>Price</th>
+      <th>Quantity</th>
+      <th>Subtotal</th>
+    </tr>
+  <% @order.item_orders.each do |item_order|%>
+    <tr>
+    <section id = "item-<%=item_order.item_id%>">
+        <td><p><%= link_to item_order.item.name, "/items/#{item_order.item_id}"%></p></td>
+        <td><p><%= item_order.item.description %></p></td>
+        <td><p><%= number_to_currency(item_order.price)%></p></td>
+        <td><p><%= item_order.item.image %></p></td>
+        <td><p><%= item_order.quantity%></p></td>
+        <td><p><%= number_to_currency(item_order.subtotal)%></p></td>
+      </section>
+    </tr>
+  <% end %>
+</table>
+
+<section id="total-quantity">
+  <p>Total Quantity: <%=@order.total_items%></p>
+</section>
+<section id="grandtotal">
+  <p>Total: <%=number_to_currency(@order.grandtotal)%></p>
+</section>
+<section id="status">
+  <p>Status: <%=@order.status.capitalize()%></p>
+</section>
+<section id="datecreated">
+  <p> <%= @order.created_at%></p>
+</section>
+<section id="dateupdated">
+  <p> <%= @order.updated_at%></p>
+</section>
```

Comment:
The id's here are a bit inconsistent. The first one is kabab case and all the rest are combined together.